src/beastie/index.ts

```diff
@@ -141,7 +142,7 @@ export default class BeastieBot {
     const response = handleStreamChange(stream, this.state.curStreamId);
 
     this.state.isStreaming = response.live;
-    this.state.curStreamId = response.streamId;
+    this.state.curStreamId = response.streamId.toString();
```

Comment:
Are you calling `toString()` because the `streamId` is of type `any`? I presume it's a real string when it is in the response, but that this call is just for satisfying the type-checker.

Perhaps the following will satisfy the type-checker without redundantly calling `.toString()`?
```suggestion
    this.state.curStreamId = response.streamId as string;
```

Reply:
streamId is a number currently, all this comes from twitchWebhooks so needs to be refactored and fixed at a later point.
Going to leave it as is now and fix it when we fix webhooks